### PR TITLE
Fix user search

### PR DESCRIPTION
**Pull request by IKarbowiak**
Fix user search - allow filtering by multiple words

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4921 Add ability to search orders by user name](https://app.clickup.com/t/2549495/SALEOR-4921)

File: saleor/graphql/account/tests/test_account.py [hunk 1]
```diff
@@ -4511,6 +4511,10 @@ def test_query_customers_with_sort(
         ({"search": "wroc"}, 1),  # city
         ({"search": "pl"}, 1),  # country
         ({"search": "+48713988102"}, 1),
+        ({"search": "Alice Kowalski"}, 1),
+        ({"search": "Kowalski Alice"}, 1),
+        ({"search": "John Doe"}, 1),
+        ({"search": "Alice Doe"}, 0),
```
Comment (L3str4nge) [inline saleor/graphql/account/tests/test_account.py:4517]: `Q(email__ilike=search_value)` I think it's missing in parametrize.

Comment (IKarbowiak) [reply saleor/graphql/account/tests/test_account.py:4517]: There is, by `mirumee.com"` as a first test.

**Review by jakubkuc — APPROVED** [review]

**Review by L3str4nge — APPROVED** [review]

**Comment by tomaszszymanski129** [comment]
@IKarbowiak could you please prepare an explain of db query this generates? Would be best on a DB with multiple users (few addresses for each). As far as I see, this generates 9 new ILIKE sql statements for each word we use in the search, could cause performance issues if we're doing a sequence scan here.

**Comment by IKarbowiak** [comment]
Closing as we decided to resolve the problem with a different solution.

**Pull request closed without merging**